### PR TITLE
Clarify documentation of DepthwiseConv2D

### PR DESCRIPTION
**Pull request by vinhill**
As discussed in [tensorflow issue 49933](https://github.com/tensorflow/tensorflow/issues/49933).

The documentation for `DepthwiseConv2D` describes the operation as: split input into individual channels, convolve each with the layer's kernel and finally stack the results. This gives the impression the same kernel is applied to all channels. But in the implementation tensorflow implementation as well as the [MobileNet Paper](https://arxiv.org/pdf/1704.04861.pdf) (see page 3 formula 3), a different kernel is applied per channel.

### CODE REVIEW

File: keras/layers/convolutional.py [hunk 1]
```diff
@@ -2258,22 +2258,24 @@ def call(self, inputs):
...
   """Depthwise 2D convolution.
 
-  Depthwise convolution is a type of convolution in which a single convolutional
-  filter is apply to each input channel (i.e. in a depthwise way).
-  You can understand depthwise convolution as being
-  the first step in a depthwise separable convolution.
-
-  It is implemented via the following steps:
+  Depthwise convolution is a type of convolution in which each input channel is
+  convolved with a different filter of depth 1 (called a depthwise kernel). You
```
Comment (fchollet) [inline keras/layers/convolutional.py:2262]: This is inaccurate: you can configure the depth, it doesn't have to be depth 1

**Review by fchollet — CHANGES_REQUESTED** [review]
Thanks for the PR

File: keras/layers/convolutional.py (hunk 1 above)
Comment (vinhill) [reply keras/layers/convolutional.py:2262]: Can you elaborate on this? I don't see an option for this in the DepthwiseConv2D class.

Comment (fchollet) [reply keras/layers/convolutional.py:2262]: That's the `depth_multiplier` argument.

Comment (vinhill) [reply keras/layers/convolutional.py]: I see what you mean and added a commit for this.

File: keras/layers/convolutional.py [hunk 2]
```diff
@@ -2290,10 +2292,9 @@ class DepthwiseConv2D(Conv2D):
       `"valid"` means no padding. `"same"` results in padding with zeros evenly
       to the left/right or up/down of the input such that output has the same
       height/width dimension as the input.
-    depth_multiplier: The number of depthwise convolution output channels
-      for each input channel.
-      The total number of depthwise convolution output
-      channels will be equal to `filters_in * depth_multiplier`.
```
Comment (fchollet) [inline keras/layers/convolutional.py:2296]: This sentence is important and should be kept

File: keras/layers/convolutional.py [hunk 3]
```diff
@@ -2258,22 +2258,24 @@ def call(self, inputs):
...
+  can understand depthwise convolution as the first step in a depthwise
+  separable convolution.
 
+  It could be implemented via the following steps:
+	
   - Split the input into individual channels.
-  - Convolve each input with the layer's kernel (called a depthwise kernel).
-  - Stack the convolved outputs together (along the channels axis).
+  - Convolve each channel with an individual depthwise kernel having 1 input
+    and `depth_multiplier` many output channels.
```
Comment (fchollet) [inline keras/layers/convolutional.py:2270]: You can remove "many"

**Review by fchollet — CHANGES_REQUESTED** [review]

File: keras/layers/convolutional.py (hunk 3 above)
Comment (vinhill) [reply keras/layers/convolutional.py:2270]:
```suggestion
    and `depth_multiplier` output channels.
```

**Review by fchollet — APPROVED** [review]
LGTM, thanks

**Pull request merged**